chia/cmds/farm_funcs.py

```diff
@@ -211,23 +212,40 @@ async def summary(rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int,
         print(f"Block rewards: {(amounts['farmer_reward_amount'] + amounts['pool_reward_amount']) / units['chia']}")
         print(f"Last height farmed: {amounts['last_height_farmed']}")
 
-    total_plot_size = 0
-    total_plots = 0
-    if all_plots is not None:
-        for harvester_ip, plots in all_plots.items():
-            if harvester_ip == "success":
-                # This key is just "success": True
-                continue
-            total_plot_size_harvester = sum(map(lambda x: x["file_size"], plots["plots"]))
-            total_plot_size += total_plot_size_harvester
-            total_plots += len(plots["plots"])
-            print(f"Harvester {harvester_ip}:")
-            print(f"   {len(plots['plots'])} plots of size: {format_bytes(total_plot_size_harvester)}")
-
-        print(f"Plot count for all harvesters: {total_plots}")
+    class PlotStats:
```

Comment:
They are not available in the nested `print_harvesters` (now `process_harvesters` after baa1fdc). I could refactor all this a bit to avoid the need of the class if that's a bad thing to do. Let me know.